website/src/controllers/api-v2/groups.js

```diff
@@ -691,8 +691,9 @@ var inviteByEmails = function(invites, group, req, res, next){
 
 api.invite = function(req, res, next){
   var group = res.locals.group;
-
-  if (req.body.uuids) {
+  if (group.memberCount + group.invites.length >= 30) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
Yup, that 30 as a variable thing sounds nice.  To be clear I mean how do I include 30 as a variable in the env.t('messageToTranslate') function and the associated templates.